pytest_catchlog.py

```diff
@@ -71,10 +72,11 @@ def catching_logs(handler, filter=None, formatter=None,
                 yield handler
 
 
-def add_option_ini(parser, option, dest, default=None, help=None):
-    parser.addini(dest, default=default,
+def add_option_ini(parser, option, dest, type=None, action=None, default=None, help=None):
+    parser.addini(dest, default=default, type=type,
```

Comment:
Ah, I see. I misinterpreted this as a default value for the argument first, dumb highlighting.
Thanks for clarifying that!